lib/ex_toolkit/roman.ex

```diff
@@ -0,0 +1,20 @@
+defmodule ExToolkit.Roman do
+  @valid_romans ["I", "V", "X", "L", "C", "D", "M"]
+
+  def is_valid_roman(string) do
+    string
+    |> String.upcase()
+    |> String.graphemes()
+    |> is_valid_roman(@valid_romans)
+  end
+
+  defp is_valid_roman([], _), do: true
+
+  defp is_valid_roman([head | tail], valid_romans) do
+    if Enum.member?(@valid_romans, head) do
+      is_valid_roman(tail, valid_romans)
+    else
+      false
+    end
+  end
```

Comment:
```suggestion
  defp is_valid_roman(graphenes) when is_list(graphenes) do
    Enum.all?(graphenes, &(&1 in @valid_romans))
  end
```

Reply:
And then you can just pipe it in the original case without the extra aux function :bulb: